submissions/Kostiantyn_Rakhmanov/dom/styles.css

```diff
@@ -0,0 +1,117 @@
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    border: 0;
+}
+
+body,
+h1,
+h2 {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 16px;
+    line-height: 24px;
+}
+
+li {
+    list-style: none;
+}
+
+a {
+    text-decoration: none;
+}
+
+@media(max-width: 1170px) {
+    .menu__items {
+        flex-direction: column;
+    }
+}
+
+.wrapper {
+    max-width: 1200px;
+    margin: 0 auto;
+    display: grid;
+    grid-template-rows: auto 1fr auto;
+    min-height: 100vh;
+}
+
+.header {
+    min-height: 50px;
+}
+
+.h1 {
+    font-size: 24px;
+    text-align: center;
+    padding: 25px;
+}
+
+.menu {
+    min-height: 50px;
+    align-items: center;
+}
+
+.menu__items {
+    display: flex;
+    justify-content: space-around;
+    padding: 5px;
+    margin: 1px;
+}
+
+.menu__item-link {
+    text-align: center;
+    display: block;
+    margin: 5px;
+    padding: 5px 10px;
+    border: 1px solid black;
+    border-radius: 10px;
+    color: black;
+    font-size: 20px;
+    line-height: 24px;
+}
+
+.menu__item-link+.menu__item-link {
+    margin-bottom: 5px;
+}
+
+.menu__item-link:hover,
+.menu__item-link:focus-within,
+.menu__item-link:active {
+    color: white;
+    background-color: black;
+}
+
+.menu__item:focus {
+    border: 1px solid red;
+}
+
+.article__title {
+    padding: 10px;
+    margin: 0 auto;
+    text-align: center;
+}
+
+.article__text {
+    padding: 15px;
+}
+
+.article__price {
+    padding: 5px;
+    font-weight: 800;
+    text-align: center;
+}
+
+.article__img {
+    padding: 5px;
+    display: block;
+    margin: 10px auto;
+    width: 100%;
+    max-height: 450px;
+    object-fit: contain;
+}
+
+.footer {
+    text-align: center;
+    font-size: 12px;
+}
```

Comment:
Also, the empty line.

Reply:
Hi!
Sorry about these lines. Looks like VSC deleled all my additional lines. Will check it in future.